docs/static/filebeat-modules.asciidoc

```diff
@@ -74,23 +74,26 @@ output {
     elasticsearch {
       hosts => "https://061ab24010a2482e9d64729fdb0fd93a.us-east-1.aws.found.io:9243"
       manage_template => false
-      index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}"
-      pipeline => "%{[@metadata][pipeline]}" <1>
+      index => "%{[@metadata][beat]}-%{[@metadata][version]}" <1>
+      pipeline => "%{[@metadata][pipeline]}" <2>
       user => "elastic"
       password => "secret"
     }
   } else {
     elasticsearch {
       hosts => "https://061ab24010a2482e9d64729fdb0fd93a.us-east-1.aws.found.io:9243"
       manage_template => false
-      index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}"
+      index => "%{[@metadata][beat]}-%{[@metadata][version]}" <1>
       user => "elastic"
       password => "secret"
     }
   }
 }
 -----
-<1> Set the `pipeline` option to `%{[@metadata][pipeline]}`. This setting
+<1> Set the `index` option to `%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}`
+if you are disabling the use of Data Streams in your configuration. Data Streams is
+enabled by default starting from 8.0.
```

Comment:
```suggestion
<1> If you are disabling the Data Streams in your configuration, set the `index` option to `%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}` instead.
Data streams are
enabled by default.
```

Reply:
Each minor revision has its own complete doc set (8.2, 8.1, 8.0, 7.17, etc.) that can easily be referenced. For that reason, we don't usually put version numbers in our content.  In the 8.3 version of the guide, saying that data streams are enabled is enough.

Reply:
Elastic standard for data streams is lower case and treated as plural.